app/views/requests/show.html.erb

```diff
@@ -3,75 +3,133 @@
 <table class="table table-fixed-columns table-striped">
   <tbody>
     <tr>
-      <th>VM Name:</td>
-      <td><%= @request.name %></td>
+      <td><strong>VM Name:</strong></td>
```

Comment:
we should probaply use the form here or make 2 seperate files (one for the admins and one for the users etc.)